NwInterface.cpp

```diff
@@ -153,6 +153,9 @@ void NwInterface::packetPollLoop() {
 	break;
       }
     }
+    else { 
+      honeyHarvesting(10);
```

Comment:
should a parameter be added to the config file instead of using _10_?

Reply:
Not that important I would say